### ISSUE TITLE
GTK 3.20 :: Toolbar/inlinetoolbar :: Button separator bug

### ISSUE THREAD

**Issue by megax** [numixproject/numix-gtk-theme#557]
![image](https://cloud.githubusercontent.com/assets/461493/18224065/eb1ac214-71c9-11e6-815b-069a76f108aa.png)
![image](https://cloud.githubusercontent.com/assets/461493/18224095/d11f5568-71ca-11e6-9a3c-c169df9aa8ba.png)
![image](https://cloud.githubusercontent.com/assets/461493/18224416/6f32b372-71d4-11e6-9566-420267462f63.png)

**Comment by megax** [comment]
@lots0logs Idea?

**Comment by lots0logs** [comment]
Which application is that?

**Comment by megax** [comment]
gedit -> print sample
nautilus -> settings

old mistakes, i forgot to send in =/ (https://github.com/numixproject/numix-gtk-theme/commit/4fb890854cb8bbeb7153e21daff5b7cd41598490 I've seen here) Base inline-toolbar code ok. A special code will be here... =/

**Comment by khurshid-alam** [comment]
@megax Did you try using separator code inside inline-toolbar? Although it should get inherited from primary-toolbar.

**Comment by megax** [comment]
@khurshid-alam I tried but ran out of idea. =/

**Comment by lots0logs** [comment]
I will try and take a closer look at this today...

**Comment by megax** [comment]
Related bug:
**Adwaita:**
![image](https://cloud.githubusercontent.com/assets/461493/18849377/02df2270-8433-11e6-8040-7bcfe2cba7ee.png)

**Numix:**
![image](https://cloud.githubusercontent.com/assets/461493/18849388/0a3de0e2-8433-11e6-8005-b2b49c7ae3bc.png)

**Comment by megax** [comment]
Maybe the problem here: https://github.com/numixproject/numix-gtk-theme/blob/2.6.5/src/gtk-3.20/scss/widgets/_button.scss#L87

**Comment by khurshid-alam** [comment]
@megax Still reproducible in 3.22?

**Comment by megax** [comment]
@khurshid-alam Yes, reproducible =/

**Comment by khurshid-alam** [comment]
I don't think we can do much here other than separating toolbar link-button code from  inline-toolbar link-button. May be we should target Gtk-4 for this.
